### PR TITLE
Cknieling/bundle installers

### PR DESCRIPTION
**Pull request by chr11115**
- bundle all installers into one installer
- avoid using c_pack abstraction layer for wix
- directly call candle/light in ant script

### CODE REVIEW

**Review by mistafunk — CHANGES_REQUESTED** [review]
Nice work overall! See inline comments for a few issues...

File: Jenkinsfile [hunk 1]
```diff
@@ -159,13 +172,30 @@ def taskBuildSerlioInstaller(cfg) {
...
 	cepl.cleanCurrentDir()
 	unstash(name: SOURCE_STASH)
-	final List deps = [ cfg.maya ]
-	deps.each { d -> papl.fetchDependency(d, cfg) }
+
+	String antArgs = ""
+
+	// fetch outputs from builds
+	dir(path: 'serlio.git'){
+		dir(path: 'build'){
```
[the remark this answers is not in the document]
Comment (chr11115) [reply Jenkinsfile]: done.

File: deploy/serlio.wxs.in [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		<Property Id="ARPPRODUCTICON">ProductIcon.ico</Property>
+		<Icon Id="ProductIcon.ico" SourceFile="@RESOURCE_FOLDER@\serlio.ico"/>
+		<WixVariable Id="WixUIBannerBmp" Value="@RESOURCE_FOLDER@\banner.png"/>
+		<WixVariable Id="WixUIDialogBmp" Value="@RESOURCE_FOLDER@\dialog.png"/>
+
+		<Directory Id='TARGETDIR' Name='SourceDir'>
+			<Directory Id='CommonAppDataFolder' Name='ProgramData'>
+				<Directory Id='AutoDeskDataFolder' Name='Autodesk'>
+					<Directory Id='ApplicationPluginsFolder' Name='ApplicationPlugins'>
+						<Directory Id='INSTALLDIR' Name='serlio'>
```
[the remark this answers is not in the document]
Comment (chr11115) [reply deploy/serlio.wxs.in]: added suffix back

File: deploy/build.xml [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+				<filter token="SRL_VERSION_BASE" value="${serlio.version.base}" />
+				<filter token="SRL_VERSION" value="${serlio.version.all}" />
+				<filter token="CPACK_PACKAGE_FILE_NAME" value="${serlio.filename}" />
+				<filter token="SRL_PKG_OS" value="${serlio.platform}" />
+			</filterset>
+		</copy>
+
+		<!-- fill package contents file -->
+		<copy file="${basedir}\PackageContents.xml.in" toFile="${deploy.dir}\PackageContents.xml">
+			<filterset>
```
[the remark this answers is not in the document]
Comment (chr11115) [reply deploy/build.xml:69]: fixed and renamed token names

File: build.ps1 [hunk 4]
```diff
@@ -46,14 +45,13 @@ function NukeTools {
...
     NukeFolder($ARCHIVE_BUILD_FOLDER)
-    NukeFolder($INSTALLER_BUILD_FOLDER)
     NukeFolder($ARTIFACTS_FOLDER)
     Write-Host ">>> Folder structure for builds destroyed." -ForegroundColor Green
 }
 
 function MakeDirs {
     # Create the desired folder structure.
-    $folders = $BUILD_FOLDER, $TOOLS_FOLDER, $NINJA_FOLDER, $CMAKE_FOLDER, $WIX_FOLDER, $ARCHIVE_BUILD_FOLDER, $INSTALLER_BUILD_FOLDER, $ARTIFACTS_FOLDER
+    $folders = $BUILD_FOLDER, $TOOLS_FOLDER, $NINJA_FOLDER, $CMAKE_FOLDER, $WIX_FOLDER, $ARCHIVE_BUILD_FOLDER, $ARTIFACTS_FOLDER
```
[the remark this answers is not in the document]
Comment (chr11115) [reply build.ps1]: removed all references to wix tools

**Review by mistafunk — CHANGES_REQUESTED** [review]
In PackageContents the tag `OnlineDocumentation` should point to `https://github.com/esri/serlio`

Also please remove all XML comments from that file including the header (it is not a cmake template anymore).

Afterwards, feel free to merge.

**Pull request merged**